java/ql/src/Security/CWE/CWE-266/IntentUriPermissionManipulation.qhelp

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>When an Android component expects a result from an Activity, <code>startActivityForResult</code> can be used.
```

Comment:
Again, should we use the words Activity, Intent and Content Providers in their capitalized form?